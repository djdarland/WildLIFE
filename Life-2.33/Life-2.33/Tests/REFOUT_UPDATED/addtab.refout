
*** Yes
> > add(1,2,3,0)
add(1,3,4,0)
add(1,4,5,0)
add(1,5,6,0)
add(1,6,7,0)
add(1,7,8,0)
add(1,8,9,0)
add(2,1,3,0)
add(2,3,5,0)
add(2,4,6,0)
add(2,5,7,0)
add(2,6,8,0)
add(2,7,9,0)
add(2,8,0,1)
add(3,1,4,0)
add(3,2,5,0)
add(3,4,7,0)
add(3,5,8,0)
add(3,6,9,0)
add(3,7,0,1)
add(3,9,2,1)
add(4,1,5,0)
add(4,2,6,0)
add(4,3,7,0)
add(4,5,9,0)
add(4,6,0,1)
add(4,8,2,1)
add(4,9,3,1)
add(5,1,6,0)
add(5,2,7,0)
add(5,3,8,0)
add(5,4,9,0)
add(5,7,2,1)
add(5,8,3,1)
add(5,9,4,1)
add(6,1,7,0)
add(6,2,8,0)
add(6,3,9,0)
add(6,4,0,1)
add(6,7,3,1)
add(6,8,4,1)
add(6,9,5,1)
add(7,1,8,0)
add(7,2,9,0)
add(7,3,0,1)
add(7,5,2,1)
add(7,6,3,1)
add(7,8,5,1)
add(7,9,6,1)
add(8,1,9,0)
add(8,2,0,1)
add(8,4,2,1)
add(8,5,3,1)
add(8,6,4,1)
add(8,7,5,1)
add(8,9,7,1)
add(9,3,2,1)
add(9,4,3,1)
add(9,5,4,1)
add(9,6,5,1)
add(9,7,6,1)
add(9,8,7,1)

*** No
> add(_A: 0,_A,_A,_A)
add(_A: 0,_B: 1,_B,_A)
add(_A: 0,_B: 2,_B,_A)
add(_A: 0,_B: 3,_B,_A)
add(_A: 0,_B: 4,_B,_A)
add(_A: 0,_B: 5,_B,_A)
add(_A: 0,_B: 6,_B,_A)
add(_A: 0,_B: 7,_B,_A)
add(_A: 0,_B: 8,_B,_A)
add(_A: 0,_B: 9,_B,_A)
add(_A: 1,_B: 0,_A,_B)
add(_A: 1,_A,2,0)
add(_A: 1,9,0,_A)
add(_A: 2,_B: 0,_A,_B)
add(_A: 2,_A,4,0)
add(2,9,_A: 1,_A)
add(_A: 3,_B: 0,_A,_B)
add(_A: 3,_A,6,0)
add(3,8,_A: 1,_A)
add(_A: 4,_B: 0,_A,_B)
add(_A: 4,_A,8,0)
add(4,7,_A: 1,_A)
add(_A: 5,_B: 0,_A,_B)
add(_A: 5,_A,0,1)
add(5,6,_A: 1,_A)
add(_A: 6,_B: 0,_A,_B)
add(6,5,_A: 1,_A)
add(_A: 6,_A,2,1)
add(_A: 7,_B: 0,_A,_B)
add(7,4,_A: 1,_A)
add(_A: 7,_A,4,1)
add(_A: 8,_B: 0,_A,_B)
add(8,3,_A: 1,_A)
add(_A: 8,_A,6,1)
add(_A: 9,_B: 0,_A,_B)
add(9,_A: 1,0,_A)
add(9,2,_A: 1,_A)
add(_A: 9,_A,8,1)

*** No
> []
*** Yes
A = @, B = @, C = @, D = @, X = add.
--1> 
*** No
> [add(9,8,7,1),add(9,7,6,1),add(9,6,5,1),add(9,5,4,1),add(9,4,3,1),add(9,3,2,1),add(8,7,5,1),add(8,6,4,1),add(8,5,3,1),add(8,4,2,1),add(7,6,3,1),add(7,5,2,1)]
*** Yes
A = @, B = @, C = @, D = @, X = add.
--1> 
*** No
> 
