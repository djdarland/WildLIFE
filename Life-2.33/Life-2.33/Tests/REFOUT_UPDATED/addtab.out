
*** Yes
> > add(1,2,3,0)
add(1,3,4,0)
add(1,4,5,0)
add(1,5,6,0)
add(1,6,7,0)
add(1,7,8,0)
add(1,8,9,0)
add(2,1,3,0)
add(2,3,5,0)
add(2,4,6,0)
add(2,5,7,0)
add(2,6,8,0)
add(2,7,9,0)
add(2,8,0,1)
add(3,1,4,0)
add(3,2,5,0)
add(3,4,7,0)
add(3,5,8,0)
add(3,6,9,0)
add(3,7,0,1)
add(3,9,2,1)
add(4,1,5,0)
add(4,2,6,0)
add(4,3,7,0)
add(4,5,9,0)
add(4,6,0,1)
add(4,8,2,1)
add(4,9,3,1)
add(5,1,6,0)
add(5,2,7,0)
add(5,3,8,0)
add(5,4,9,0)
add(5,7,2,1)
add(5,8,3,1)
add(5,9,4,1)
add(6,1,7,0)
add(6,2,8,0)
add(6,3,9,0)
add(6,4,0,1)
add(6,7,3,1)
add(6,8,4,1)
add(6,9,5,1)
add(7,1,8,0)
add(7,2,9,0)
add(7,3,0,1)
add(7,5,2,1)
add(7,6,3,1)
add(7,8,5,1)
add(7,9,6,1)
add(8,1,9,0)
add(8,2,0,1)
add(8,4,2,1)
add(8,5,3,1)
add(8,6,4,1)
add(8,7,5,1)
add(8,9,7,1)
add(9,3,2,1)
add(9,4,3,1)
add(9,5,4,1)
add(9,6,5,1)
add(9,7,6,1)
add(9,8,7,1)

*** No
> add(0,0,0,0)
add(0,1,1,0)
add(0,2,2,0)
add(0,3,3,0)
add(0,4,4,0)
add(0,5,5,0)
add(0,6,6,0)
add(0,7,7,0)
add(0,8,8,0)
add(0,9,9,0)
add(1,0,1,0)
add(1,1,2,0)
add(1,9,0,1)
add(2,0,2,0)
add(2,2,4,0)
add(2,9,1,1)
add(3,0,3,0)
add(3,3,6,0)
add(3,8,1,1)
add(4,0,4,0)
add(4,4,8,0)
add(4,7,1,1)
add(5,0,5,0)
add(5,5,0,1)
add(5,6,1,1)
add(6,0,6,0)
add(6,5,1,1)
add(6,6,2,1)
add(7,0,7,0)
add(7,4,1,1)
add(7,7,4,1)
add(8,0,8,0)
add(8,3,1,1)
add(8,8,6,1)
add(9,0,9,0)
add(9,1,0,1)
add(9,2,1,1)
add(9,9,8,1)

*** No
> []
*** Yes
A = @, B = @, C = @, D = @, X = add.
--1> 
*** No
> [add(9,8,7,1),add(9,7,6,1),add(9,6,5,1),add(9,5,4,1),add(9,4,3,1),add(9,3,2,1),add(8,7,5,1),add(8,6,4,1),add(8,5,3,1),add(8,4,2,1),add(7,6,3,1),add(7,5,2,1)]
*** Yes
A = @, B = @, C = @, D = @, X = add.
--1> 
*** No
> 
